### DISCUSSION TITLE
Search in Playlist

### DISCUSSION THREAD

**Discussion by jwedenig** [iina/iina#5296] [General]
Sorry, i cannot find it, so how can i search in my m3u, because it is very large?
Is there a function to do that?
Thanks

**Comment by low-batt** [comment]
A very common question because the search is hiding under the `Help` menu. That search covers many places such as menus, settings and the playlist. The playlist is populated in the background, so if you start searching right away the search might not find anything.

I recall there is an open issue about search being very slow with large (50,000) playlists. And there was an issue about certain settings not being picked up in the search. I know that has been fixed, but I don't think that fix is in IINA 1.3.5.
![search](https://github.com/user-attachments/assets/8d262082-79b2-4210-b5d5-3ed8005c905f)

**Reply by jwedenig** [reply]
Aha, thanks
